examples/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM rust:1.35 as build
+
+WORKDIR /usr/src/
+
+RUN apt-get update && apt-get install -y \
+    git \
+    cmake \
+    golang \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN git clone --recurse-submodules --depth 1 https://github.com/cloudflare/quiche
+
+RUN cd quiche && cargo build --release --examples
+
+FROM debian:latest
+RUN apt-get update && apt-get install -y \
+    ca-certificates \
+ && rm -rf /var/lib/apt/lists/*
+RUN update-ca-certificates
+WORKDIR /usr/src/
```

Comment:
Installing binaries into /usr/src doesn't looks a good practice. `/usr/local/quiche/bin` or `/usr/local/bin` looks good to me. Also we can use `cargo install ...` instead of copy.

Reply:
Updated to COPY to `/usr/local/quiche/bin` and add that to path.

AFAICT the `cargo install` command won't help much here because we are copying from one container to the other. So even if we installed on the build machine, we'd still need the COPY. Separate COPYs create individual layers that make speed up docker build times if changes modify examples in isolation.

Reply:
> Updated to COPY to `/usr/local/quiche/bin` and add that to path.
In a new commit it's `/usr/local/bin/quiche`, not `/usr/local/quiche/bin`.

Reply:
oops, I fixed it now